regression-test/suites/nereids_function_p0/scalar_function/T.groovy

```diff
@@ -45,6 +45,10 @@ suite("nereids_scalar_fn_T") {
 	qt_sql_timestamp_DateTime_notnull "select timestamp(kdtm) from fn_test_not_nullable order by kdtm"
 	qt_sql_timestamp_DateTimeV2 "select timestamp(kdtmv2s1) from fn_test order by kdtmv2s1"
 	qt_sql_timestamp_DateTimeV2_notnull "select timestamp(kdtmv2s1) from fn_test_not_nullable order by kdtmv2s1"
+	qt_sql_timestamp_DateTime "select timestamp(kdtm,'Asia/Bangkok') from fn_test order by kdtm"
+	qt_sql_timestamp_DateTime_notnull "select timestamp(kdtm,'Asia/Bangkok') from fn_test_not_nullable order by kdtm"
+	qt_sql_timestamp_DateTimeV2 "select timestamp(kdtmv2s1,'Asia/Bangkok') from fn_test order by kdtmv2s1"
+	qt_sql_timestamp_DateTimeV2_notnull "select timestamp(kdtmv2s1,'Asia/Bangkok') from fn_test_not_nullable order by kdtmv2s1"
```

Comment:
If the function supports two column arguments, we'd better test this case, that is, `timestamp(column1, coulumn2)`.

Also, the tag for sentence(like `qt_sql_timestamp_DateTimeV2`) should be different from the any other's.